### PR TITLE
Oscilloscope component linked to device, single channel tested

### LINKED ISSUE #242: Implement Oscilloscope graph and link app to device
## Expected behaviour
- The oscilloscope graph shall be able to pick up data from the device and represent it

## Actual behaviour 
- The oscilloscope should be able to connect to the device and draw data. The graph should be able to plot the data hence fetched.

## Would you like to work on it?
Yes

### PR DESCRIPTION
**Pull request by AakashMallik**
Fixes #242

## Changes
- [x] Added Graph to Oscilloscope
- [x] Tested it for 10k samples, for 10usec tg and 100 milsec delay
- [x] Fixed minor UI issues with Settings scroll bar
- [x] Updated **electron-load-balancer** to 1.0.5

### CODE REVIEW

**Comment by AakashMallik** [comment]
@fragm3 @CloudyPadmal @mariobehling A review would be nice. For practical testing of this code with device be sure to run npm install again as I have updated the version of my library I wrote. Had a few bugs.

File: src/scripts/bridge.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            x, y = self.device.capture1(
+                'CH1', self.number_of_samples, self.time_gap)
+            output = {'ch1': y.tolist()}
+            print(json.dumps(output))
+            sys.stdout.flush()
+            time.sleep(0.001 * self.delay)
+
+    def get_two_channel_data(self):
+        while self.isReading:
+            x, y = self.device.capture1(
```
Comment (fragm3) [inline src/scripts/bridge.py]: capture2 instead of capture1

File: src/screen/Oscilloscope/Oscilloscope.js [hunk 2]
```diff
@@ -1,20 +1,68 @@
...
+		});
+	}
+
+	componentWillUnmount() {
+		ipcRenderer.removeAllListeners('TO_RENDERER_OSC_DATA');
+	}
+
+	onToggleRead = event => {
+		const { isReading, activeChannels } = this.state;
+		this.setState({
```
Comment (fragm3) [inline src/screen/Oscilloscope/Oscilloscope.js]: Use: 
```
this.setState(prevState => ({
  isReading: !prevState. isReading
}));
```

File: src/scripts/bridge.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
+from PSL import sciencelab
```
Comment (fragm3) [inline src/scripts/bridge.py]: Can `bridge.py` be renamed to something more specific?

File: src/scripts/bridge.py (hunk 1 above)
Comment (AakashMallik) [reply src/scripts/bridge.py]: Yes, this is yet to be implemented, so I left it intentionally.

File: src/scripts/bridge.py (hunk 3 above)
Comment (AakashMallik) [reply src/scripts/bridge.py]: As of now, I think we can go for a single python file, which acts as a **bridge**. But we can later change it if we plan to go for multiple files, or we could change it to a more sensible name I guess. I'll try to find something more specific.

**Comment by AakashMallik** [comment]
@fragm3 Made the changes... :lollipop:

File: src/screen/Oscilloscope/Oscilloscope.js (hunk 2 above)
Comment (AakashMallik) [reply src/screen/Oscilloscope/Oscilloscope.js]: Done

**Comment by AakashMallik** [comment]
@CloudyPadmal @fragm3 @mariobehling Don't merge this as of now. I have implemented a newer architecture and merging this would cause merge conflicts for me. Let @fragm3 and I test that version completely and then I'll close this PR if needed.

**Pull request closed without merging**